Sources/PointFree/SVG.swift

```diff
@@ -1,227 +0,0 @@
-import Foundation
```

Comment:
I'm deleting this file because somehow there was also a copy of this file in the Views framework. maybe when we were modularizing we forgot to delete this...